django_app/redbox_app/redbox_core/models.py

```diff
@@ -734,7 +734,7 @@ class ChatRoleEnum(models.TextChoices):
 class Citation(UUIDPrimaryKeyBase, TimeStampedModel):
     class Origin(models.TextChoices):
         WIKIPEDIA = "Wikipedia", _("wikipedia")
-        USER_UPLOADED_DOCUMENT = "USER UPLOADED DOCUMENT", _("user uploaded document")
+        USER_UPLOADED_DOCUMENT = "UserUploadedDocument", _("user uploaded document")
```

Comment:
are you sure you want this change? whilst i agree that this is better it will need:
1. a new migration to losen the field to just be a text field
2. a new data migration to change all existing instances of "USER UPLOADED DOCUMENT" to "UserUploadedDocument"
3. this migration to set the new enums 
4. a test to show that the data-migration is working

Reply:
I did wonder about this. I think we need a consistent way of mapping the 'creatortype' field set by tools to a Citation.Origin in Django so we avoid this kind of thing happening a lot. It seems like it makes sense to do now as we haven't gone to prod yet? Maybe the best balance is to take this approach of matching the strings, make a test for that but revert this one back to what it was to avoid the migration?

Reply:
I agree with your proposed change, but please can we do this in a separate PR because:

1. the change is big - and this PR is already substantial 
2. i would like to test this in dev, but its challenging to test anything that involves a migration in dev given that they are not reversible
?